src/zkbob/ZkBobPool.sol

```diff
@@ -171,6 +227,27 @@ abstract contract ZkBobPool is IZkBobPool, EIP1967Admin, Ownable, Parameters, Ex
         emit UpdateRedeemer(address(_redeemer));
     }
 
+    /**
+     * @dev Updates grace period duration.
+     * Callable only by the contract owner / proxy admin.
+     * @param _gracePeriod new grace period duration.
+     */
+    function setGracePeriod(uint64 _gracePeriod) external onlyOwner {
+        gracePeriod = _gracePeriod;
+        emit UpdateGracePeriod(_gracePeriod);
+    }
+
+    /**
+     * @dev Updates minimal fee required to be reserved for the prover who will submit the tree update proof.
+     * Callable only by the contract owner / proxy admin.
+     * @param _minTreeUpdateFee new minimal fee.
+     */
+    function setMinTreeUpdateFee(uint64 _minTreeUpdateFee) external onlyOwner {
```

Comment:
There is a censorship vector here, setting a too large min fee will prevent everyone from submitted transactions. Maybe we should make it immutable?

Reply:
Well, immutable values still can be changed during contracts upgrade, which are usually put behind timelocks. Of course we can use the time lock for `owner` as well, not just the upgradeability owner. I think it's fine to keep it this way for the upcoming release. 

One other interesting idea I had was to allow only slow changes in min fee, e.g. we can allow only +-20% min fee change per 24h.

Reply:
Okay, let's keep it as is for now. Yes, it is a good idea. I think we'll need to introduce more holistic logic to manage this value, especially if we implement batching.